### PR TITLE
[T2A3][F11 - A04]ChiangWengKiat

### PR DESCRIPTION
**Pull request by wengkiat**
I edited in the old branch by mistake, hence, unable to commit in the new branch.

Reduce SLAP of method

### CODE REVIEW

**Comment by wengkiat** [comment]
Ready for review

**Comment by jeffryhartanto** [comment]
Change the title according to my comment in #295. Make sure you use the correct title next time.

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -200,7 +200,18 @@
      */
     public static void main(String[] args) {
         showWelcomeMessage();
```
Comment (cmkumar87) [inline src/seedu/addressbook/AddressBook.java:202]: Think about other constructs such as variables that also contribute to abstraction.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -200,7 +200,18 @@
...
-        processProgramArgs(args);
+        if (args.length >= 2) {
```
Comment (cmkumar87) [inline src/seedu/addressbook/AddressBook.java:203]: think about other constructs such as variables that also contribute towards abstraction.

**Pull request closed without merging**